std/assembly/polyfills.ts

```diff
@@ -1,39 +1,46 @@
 export function bswap<T extends number>(value: T): T {
   if (isInteger<T>()) {
+    if (sizeof<T>() == 1) {
+      return value;
+    }
     if (sizeof<T>() == 2) {
-      return <T>((value << 8) | ((value >>> 8) & <T>0x00FF));
+      return <T>((value << 8) | ((value >>> 8) & 0x00FF));
```

Comment:
Shouldn't this work already for 16-bit types?

```suggestion
      return <T>((value << 8) | (value >>> 8));
```

Reply:
yeah

Reply:
Unfortunately it's wrong. Tests fail

Reply:
Hmm, strange. Makes me wonder if this is a deeper codegen problem.